From: Kevin Roy <kiniou@gmail.com>
Date: Mon, 24 Feb 2014 17:43:49 +0100
Subject: blender_thumbnailer

* Move thumbnailer script to an appropriate destination
* Add blender.thumbnailer file
* Change to python 3.3 since lintian is messing around with that

Signed-off-by: Matteo F. Vescovi <mfvescovi@gmail.com>
---
 blender.thumbnailer                | 4 ++++
 release/bin/blender-thumbnailer.py | 2 +-
 source/creator/CMakeLists.txt      | 2 +-
 3 files changed, 6 insertions(+), 2 deletions(-)
 create mode 100644 blender.thumbnailer

diff --git a/blender.thumbnailer b/blender.thumbnailer
new file mode 100644
index 0000000..05d4fae
--- /dev/null
+++ b/blender.thumbnailer
@@ -0,0 +1,4 @@
+[Thumbnailer Entry]
+TryExec=/usr/share/blender/scripts/blender-thumbnailer.py
+Exec=/usr/share/blender/scripts/blender-thumbnailer.py %i %o
+MimeType=application/x-blender;
diff --git a/release/bin/blender-thumbnailer.py b/release/bin/blender-thumbnailer.py
index 91aeb7a..10b292f 100755
--- a/release/bin/blender-thumbnailer.py
+++ b/release/bin/blender-thumbnailer.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 # ##### BEGIN GPL LICENSE BLOCK #####
 #
diff --git a/source/creator/CMakeLists.txt b/source/creator/CMakeLists.txt
index 4d12040..a99a9c6 100644
--- a/source/creator/CMakeLists.txt
+++ b/source/creator/CMakeLists.txt
@@ -453,7 +453,7 @@ if(UNIX AND NOT APPLE)
 		)
 		install(
 			PROGRAMS ${CMAKE_SOURCE_DIR}/release/bin/blender-thumbnailer.py
-			DESTINATION ${CMAKE_INSTALL_PREFIX}/bin
+			DESTINATION ${CMAKE_INSTALL_PREFIX}/share/blender/scripts
 		)
 		install(
 			FILES ${CMAKE_SOURCE_DIR}/doc/manpage/blender.1
